certificate-service/app/src/main/java/se/inera/intyg/certificateservice/application/certificate/dto/GetCertificateInternalXmlResponse.java

```diff
@@ -15,6 +16,7 @@ public class GetCertificateInternalXmlResponse {
   String certificateType;
   String unitId;
   String xml;
+  Revoked revoked;
```

Comment:
Use a DTO for the "revoked"-info, now it is a domain object.

Reply:
Changed the DTO so that it matches the miro design:

```java
@JsonDeserialize(builder = RevokedDTOBuilder.class)
@Value
@Builder
public class RevokedDTO {

  String reason;
  String message;
  LocalDateTime revokedAt;
  Staff revokedBy;

  @JsonPOJOBuilder(withPrefix = "")
  public static class RevokedDTOBuilder {

  }
}
```